plone/app/locales/locales/de/LC_MESSAGES/plone.po

```diff
@@ -647,6 +647,7 @@ msgid "Add-ons"
 msgstr "Erweiterungen"
 
 #: plone.app.iterate/plone/app/iterate/browser/diff.pt:168
+#, fuzzy
```

Comment:
@MrTango suddenly some messages are marked as fuzzy, is this a wanted change?

Reply:
no, i just used the weblate ui to translate some strings. I don't know how and why that changed.

Reply:
If you check the "it needs editing" checkbox in the weblate UI, it marks the messages as `fuzzy`

Reply:
i don't recall doing that, but good to know, thx.

Reply:
I have resolved them directly in Weblate. Thanks!